index.html

```diff
@@ -2653,6 +2653,7 @@ <h3 id="applying">
               </li>
             </ol>
           </li>
+          <li><a>Trim</a>(<var>value</var>) and convert to lower-case</li>
```

Comment:
Also I notices that the links at the end of Trim are not dead links @marcoscaceres